chrome_extension/background.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
```

Comment:
Linting rules are flagging this due to a lacking license header, seems we should add it?

Reply:
Done.